engine/src/main/java/org/terasology/persistence/typeHandling/reflection/SerializationSandbox.java

```diff
@@ -46,7 +33,9 @@
      * @param <T>      The base type whose subtype needs to be identified.
      * @return The unique identifier for {@code subType}.
      */
-    <T> String getSubTypeIdentifier(Class<? extends T> subType, Class<T> baseType);
+    default <T> String getSubTypeIdentifier(Class<? extends T> subType, Class<T> baseType) {
+        return subType.getName();
```

Comment:
I think there are no longer any implementations of this that use `baseType` at all.

Should we change the interface and remove that parameter entirely?

If so, is this still a subtypeIdentifier, or is it just its Identifier?

Reply:
Probably just keep simplifying, if we need something like this in the future we could put it back on. Might still be good to hear from @eviltak 

Additionally serialization and/or the whole ECS likely would majorly change when reintegrated as libraries anyway, so we don't have to aim for perfection right now